package.json

```diff
@@ -69,6 +69,8 @@
     "karma-ng-html2js-preprocessor": "^1.0.0",
     "karma-webpack": "^4.0.0-rc.3",
     "loader-utils": "^2.0.0",
+    "lighthouse": "^5.6.0",
```

Comment:
There's no need to add lighthouse here since we are only using lighthouse CI.

Reply:
Oh ok, Umm so I ran yarn install. Should I also uninstall it from the yarn.lock? If so do you know how?

Reply:
You could use `yarn remove <packagename>` to uninstall packages and that will update `package.json` and `yarn.lock`.

Reply:
Done